### ISSUE TITLE
[Operation Solar Raid] Solis

### ISSUE THREAD

**Issue by danielwerg** [marcopixel/r6operators#38]
## Character

- Name: Solis
- Role: Defender
- ORG: AFEAU
- SQUAD: GHOSTEYES

## Ratings:

- Speed: 2
- Health: 2
- Difficulty: 3

## Metadata:

- Sex: f
- Country: co
- Season: Y7S4
- Height: 1.66m
- Weight: 65kg

## Biography

- Real name: Ana Valentina Díaz
- Birthplace: Zipaquirå, Colombia

## Icon

<details>
<summary>Click here to show the icon</summary>

![solis icon](https://user-images.githubusercontent.com/35052399/203335286-09e54d91-7d9f-4297-bdb7-7779fa6af6bf.png)

</details>

## Reference

<details>
<summary>Health/Speed/Difficulty</summary>

![solis ug ratings](https://user-images.githubusercontent.com/35052399/203308911-5da9af0d-955a-4b85-a57a-a5fc33023c70.jpg)

</details>

<details>
<summary>Bio</summary>

![solis bio](https://user-images.githubusercontent.com/35052399/203308938-a4b1a714-5d2e-4b32-acd9-d9211bb9ba3e.jpg)

</details>

**Comment by danielwerg** [comment]
My take on Solis Icon:

<details><summary>Show</summary>

```svg
<svg
  xmlns="http://www.w3.org/2000/svg"
  viewBox="0 0 350 350"
>
  <rect x="45" y="45" width="260" height="260" opacity="0.4" />
  <path d="M37 38 124 103 113 73H122L131 82V73H210L206 105 230 73H240L227 103 313 38 249 124 278 112V122L262 140H278V208H252L279 230V240L248 226 313 312 227 248 240 278H229L212 263V278H142L158 233 122 278H112L125 248 37 312 103 227 73 240V230L98 215H73V143H101L73 121V111L104 125Z" fill="#e7411f" />
  <path d="M157 7H161L171 25V73L154 91 171 107V117L176 113 181 117 194 104 180 91V54L189 37H192L202 76 218 92V118L208 127V132L213 136H232L243 125 H278L 295 143 345 156V160L327 170H278L261 153 244 170H233V180L247 194 260 180H299L315 188V191L275 201 261 217H234L225 208 215 215V231L226 243V275L207 295 195 343H191L181 325V278L199 260 181 243V233L176 236 171 233 158 246 171 259V297L163 314H160L150 274 135 260V233L143 224V218L139 215H119L108 226H77L57 207 8 194V190L27 180 73 180 92 198 109 180 119 180V172L105 157 92 170H54L39 163V160L77 149 93 133H118L127 142 137 137V119L125 108V76L144 56Z" fill="#0e0e0e" />
  <path d="M132 77 166 110V127L176 120 186 126V120L202 104 186 88V54L190 48 198 80 212 94V115L203 123V135L212 142H235L247 130H275L241 165H228V183L247 201 263 185H297L303 190 272 197 258 212H236L215 190 176 213 143 194 143 186 176 205 216 182V153L176 129 138 151 133 145 143 139V117L132 105Z" fill="#ffffff" />
  <path d="M210 157V165L176 145 135 168V198L176 221 214 199 220 205 210 211V234L221 245V274L187 240V224L176 231 166 224V231L150 246 166 262V295L163 302 154 271 140 257V235L149 226V215L140 209H117L106 220H77L111 186H125V169L105 149 89 165H56L49 161 79 153 94 140H116L137 160 176 137Z" fill="#ffffff" />
  <path d="M160 17 166 27V70L155 82 135 73 149 60ZM278 135 292 149 334 159 324 165H281L270 154ZM187 280 199 269 216 277 201 291 191 333 187 324ZM18 191 28 185H71L83 197 74 216 60 203Z" fill="#ffffff" />
  <path d="M182 157 209 173A3 3 0 01209 178L182 193 195 175 190 169 182 174ZM170 157 157 175 170 193 143 178A3 3 0 01143 173Z" fill="#ffffff" />
</svg>

```

</details>

**Comment by SwiftCODA** [comment]
I think that looks pretty good but we will see what the master icon enhancer says.

**Comment by marcopixel** [comment]
Thank you for your help @danielwerg, your icon was pretty good and i didn't need to modify much except positioning it inside the template and adding the eye in the middle :) 

This issue will be closed with the Release [`2.3.0`](https://github.com/marcopixel/r6operators/releases/tag/2.3.0).

**Comment by danielwerg** [comment]
@marcopixel huh, it already has an eye :D
illustrator doesn't parse it for some reason.
